tests/python/unittest/test_operator.py

```diff
@@ -2109,6 +2109,10 @@ def test_reshape_new(src_shape, shape_args, reverse, dst_shape):
                 'Output Shape = %s' %(str(holdout_src_shape), str(shape_args), str(reverse),
                                       str(dst_shape), str(output_shape[0]))
 
+    def test_invalid_reshape():
```

Comment:
Can you remove the test_ prefix? It is reserved for top level functions that are being tested